template/src/project_name/main/settings.py

```diff
@@ -144,3 +146,22 @@
         },
     },
 }
+
+# Sentry
+sentry_sdk.init(
+    dsn=env("SENTRY_DSN", default=None),
+    environment=env("SENTRY_ENV", default="production"),
+    release=GIT_COMMIT_HASH,
```

Comment:
Suggestion (non-blocking): Use `GIT_COMMIT_COUNT`

I'm in two minds about this one. Using the `hash` will allow you to tie an alert back to a specific commit. It is also effectively a random number, there is no (simple) way to tell if one hash comes before another. The purpose of the `GIT_COMMIT_COUNT` is to provide an ever increasing number so that it is easy to tell which comes after which.

Reply:
I think I'm softly against this, at least for now as while I understand the thinking and agree with the issue, I think in the long-run it's actually harder to map that count to a specific version of a codebase (with a commit SHA I can just give that directly to `git` in a number of ways), and Sentry itself uses this value when linked with the `git` repository which I don't think will work if we're using a count

i.e. Sentry will be able to tell us if a particular release comes before or another another, among other cool features

Reply:
ok, I'm fine with that if Sentry uses it to link to the git release then lets leave it as the git hash.